name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/AbstractPDFExtractor.java

```diff
@@ -133,6 +127,16 @@ public String getFilterExtension()
             return Collections.emptyList();
         }
     }
+    
+    protected List<Item> parseDocumentTypes(List<DocumentType> documentTypes, String filename, String text) {
```

Comment:
Du meinst Zeilen 106-112 zu 132-137? Die sind auch gleich - das Standardverhalten ist ja identisch geblieben. ich habe es nur in die Methode parseDocumentTypes ausgelagert, um es für den OnVista Parser überschreiben zu können.